include/library/spdm_return_status.h

```diff
@@ -1,12 +1,16 @@
 /**
  *  Copyright Notice:
- *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  Copyright 2021-2024 DMTF. All rights reserved.
  *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
  **/
 
 #ifndef SPDM_RETURN_STATUS_H
 #define SPDM_RETURN_STATUS_H
 
+#ifdef __cplusplus
```

Comment:
Is there any downside of adding this, even though it's not needed? I find having this `extern c` blocks consistently across all headers give me a peace of mind. If developers were to add functions, the new apis would be readily available to use in cpp.

Reply:
> Is there any downside of adding this, even though it's not needed?

I'd like its inclusion to be intentional when needed. Presumably this will lead to C++ compilation in CI/CD to catch any compatibility issues. See below.

Reply:
okay gotcha. I've reverted my change in this file.